public/video-ui/src/components/Videos/Videos.js

```diff
@@ -37,12 +36,27 @@ class Videos extends React.Component {
     return (this.props.videos.map((video) => <VideoItem key={video.id} video={video} />));
   }
 
+  renderMoreLink() {
+    if(this.props.limit === -1) {
+      return false;
+    }
```

Comment:
When is this limit set to -1? Should we also render something else when there is nothing more to show?